src/components/Dropdown/Dropdown.tsx

```diff
@@ -71,7 +72,7 @@ export const Dropdown: FC<DropdownProps> = React.memo(
             useFloating({
                 placement,
                 strategy: positionStrategy,
-                middleware: [flip(), fOffset(offset), shift()],
+                middleware: [fOffset(offset), flip(), shift()],
```

Comment:
One interaction bug I found, and it's minor, is the direction of the animation isn't flipped. If this change could also address it that would be good, but not a blocker as the current bug is more impactful to the UX. We may address this in a future change if needed.

https://user-images.githubusercontent.com/99700808/195420954-14dfd30d-55d3-4722-bb52-a653dd1da2c1.mp4

Reply:
Perhaps this might do the trick --> https://floating-ui.com/docs/flip#flipalignment

Reply:
Let's address this in another ticket/PR so as to not block on this. Let's also keep `fOffset` as the first argument to follow convention as mentioned in the docs.